README.md

```diff
@@ -1 +1,93 @@
 # EarthML
+
+**Machine learning and visualization in Python for Earth science**
+
+EarthML is an
+[open-source](https://github.com/ioam/holoviews/blob/master/LICENSE.txt)
+Python library for exploratory machine learning research in Earth
+science. EarthML is designed to be a lightweight project with the aim of
+illustrating how other, freely available, general-purpose open source
+and [PyViz](http://pyviz.org) projects may be used to solve problems in
+this domain:
+
+> -   [Bokeh](http://bokeh.pydata.org): Interactive
+>     browser-based plotting.
+> -   [HoloViews](http://holoviews.org): Easy construction of Bokeh
+>     plots for datasets.
+> -   [GeoViews](http://geoviews.org): HoloViews with
+>     earth-specific projections.
+> -   [Datashader](https://github.com/bokeh/datashader): Rendering large
+>     datasets into images for display in browsers.
+> -   [XArray](http://xarray.pydata.org): Processing gridded
+>     data structures.
+> -   [Dask](http://dask.pydata.org/en/latest/): Parallelism and
+>     performance at scale.
+
+A major goal of this website is to demonstrate examples of how Python
+tools may be used to perform machine learning tasks in the earth
+sciences. A number of such examples are available in the
+[Topics](topics) section:
+
+<div>
+<div >
+  <a href="http://earthml.pyviz.org/topics/trees.html">
```

Comment:
This link is not quite right.

Reply:
Thanks. I think this is part of how `nbsite` is being used.